Makefile

```diff
@@ -94,6 +96,7 @@ C_CARGO_TARGET_DIR = target/release
 C_LIB_CARGO_BASENAME=libqiskit_cext
 ifeq ($(OS), Windows_NT)
 	C_DYLIB_EXT=dll
+	C_LIB_CARGO_BASENAME=qiskit_cext
```

Comment:
Interesting that this worked before, because I agree on windows the library should just be called `qiskit_cext.dll`.. At this point, I think it would be cleaner to not have the variables `C_DYLIB_EXT` and `C_LIB_CARGO_BASENAME` at all and just define `C_LIB_CARGO_FILENAME` directly in the if-else block here.